README.md

```diff
@@ -83,6 +83,18 @@ System.out.println("Balances: " + balances.stream()
 // End session
 currencyCloud.endSession();
 ```
+In order to customise the HTTP client use the HTTP client configuration builder e.g.,
+```Java
+CurrencyCloudClient currencyCloud = new CurrencyCloudClient(
+    CurrencyCloudClient.Environment.demo, 
+    "<your login id>",
+    "<your API key>",
+    CurrencyCloudClient.HttpClientConfiguration.builder()
+        .httpConnTimeout(3000)
+        .httpReadTimeout(45000)
```

Comment:
Are these the same values as the previous defaults?

Reply:
No, this is an example in the readme file. The defaults are defined here: https://github.com/CurrencyCloud/currencycloud-java/blob/master/src/main/resources/rescu.properties